### PR TITLE
Rewrite the README page

### PR DESCRIPTION
**Pull request by zupo**
Refs https://github.com/niteoweb/pareto/issues/34

### CODE REVIEW

**Comment by zupo** [comment]
This is how it looks: https://github.com/ParetoSecurity/pareto-mac/blob/a0936ed889e74838cca15ed60bc76204ee51a682/README.md

File: README.md [hunk 1]
```diff
@@ -1,14 +1,61 @@
...
+</p>
 
-More info and newsletter signup at https://paretosecurity.app/.
 
-![image](https://user-images.githubusercontent.com/239513/126497099-1b89a8ef-68f2-4aae-9f05-f124748685d2.png)
+## Peace of Mind
+
+The simplest security is the most important. 80% of hacks are caused by 20% of common preventable mistakes. Pareto Security is a Menu Bar app that quietly runs in the background and prevents security facepalms, so you can focus on more important things in life.
+
+- You **forgot to enable FileVault** disk encryption, then lost your macbook and now a stranger has all your photos and documents?
```
Comment (dz0ny) [inline README.md:34]:
```suggestion
- You **forgot to enable FileVault** disk encryption, then lost your MacBook and now a stranger has all your photos and documents?
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,61 @@
...
+- You needed to **turn off Gatekeeper** to be able to install an app you found on the Internet, then failed to turn it back on again and got malware?
+- You **turned off firewall** to debug a router, then did not remember to turn it on and accidentally exposed your local webserver to the entire Starbucks network?
```
Comment (dz0ny) [inline README.md:36]:
```suggestion
- You **turned off firewall** to debug a router, then did not remember to turn it on and accidentally exposed your local webserver to the entire Starbucks or hotel network?
```

**Review by dmurko — APPROVED** [review]
I like the examples. We should incorporate them into the sales page in the next iteration.

**Pull request merged**